server/plugin/api.go

```diff
@@ -555,20 +556,20 @@ func (p *Plugin) getConnected(c *Context, w http.ResponseWriter, r *http.Request
 	config := p.getConfiguration()
 
 	type ConnectedResponse struct {
-		Connected         bool               `json:"connected"`
-		GitHubUsername    string             `json:"github_username"`
-		GitHubClientID    string             `json:"github_client_id"`
-		EnterpriseBaseURL string             `json:"enterprise_base_url,omitempty"`
-		Organization      string             `json:"organization"`
-		UserSettings      *UserSettings      `json:"user_settings"`
-		PluginSettings    ClientSafeSettings `json:"plugin_settings"`
+		Connected         bool                   `json:"connected"`
+		GitHubUsername    string                 `json:"github_username"`
+		GitHubClientID    string                 `json:"github_client_id"`
+		EnterpriseBaseURL string                 `json:"enterprise_base_url,omitempty"`
+		Organization      string                 `json:"organization"`
+		UserSettings      *UserSettings          `json:"user_settings"`
+		PluginSettings    map[string]interface{} `json:"plugin_settings"`
```

Comment:
`map[string]interface{}` is used as `ClientSafeSettings` can't be send as a websocket event.

Reply:
Another silly question as I am still new to this plugin. how is this prevented from sending out as a web socket event ?

Reply:
`PublishWebSocketEvent` accepts an `map[string]interface{}` as an argument. `gorp` is then used for encoding the request to the MM server. But `gorp` doesn't know how to handle arbitrary structs like `ClientSafeSettings`. Hence a `map[string]interface{}` needs to be used as `gorp` knows how to encode that.